### PR TITLE
feat: add platform-app package and adjust codeowners

### PR DESCRIPTION
**Pull request by julianiff**
- Adds the Platform App package to the brand-sdk

### CODE REVIEW

File: packages/platform-app/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Platform App Types
+
+Provides typing for Platform Apps
```
Comment (SamuelAlev) [inline packages/platform-app/README.md:3]: Can you add a bit more info like in https://github.com/Frontify/brand-sdk/blob/main/packages/guideline-blocks-settings/README.md ?

File: packages/platform-app/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
+        "module": "ESNext",
+        "moduleResolution": "Node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "noEmit": true,
+        "jsx": "react-jsx"
+    },
+    "include": ["src/**/*", "tests/**/*"],
```
Comment (SamuelAlev) [inline packages/platform-app/tsconfig.json:19]:
```suggestion
    "include": ["src/**/*"],
```

File: packages/platform-app/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "vitest": "^0.34.6",
+        "react": "^18",
+        "react-dom": "^18"
+
+    },
+    "dependencies": {
+        "@frontify/sidebar-settings": "^0.8.0"
+    },
+    "peerDependencies": {
+        "@frontify/app-bridge": "^3 || ^3.1.0"
```
Comment (SamuelAlev) [inline packages/platform-app/package.json:49]:
```suggestion
        "@frontify/app-bridge": "^3.0.0 || ^4.0.0-alpha.0",
        "react": "^18",
        "react-dom": "^18"
```

File: packages/platform-app/package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        "typescript": "^5.3.2",
+        "vite": "^5.0.4",
+        "vite-plugin-dts": "^3.6.3",
+        "vitest": "^0.34.6",
+        "react": "^18",
+        "react-dom": "^18"
+
+    },
+    "dependencies": {
+        "@frontify/sidebar-settings": "^0.8.0"
```
Comment (SamuelAlev) [inline packages/platform-app/package.json:46]:
```suggestion
        "@frontify/sidebar-settings": "workspace:^"
```

File: packages/platform-app/package.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        "eslint": "^8.54.0",
+        "eslint-plugin-notice": "^0.9.10",
+        "prettier": "^3.1.0",
+        "typescript": "^5.3.2",
+        "vite": "^5.0.4",
+        "vite-plugin-dts": "^3.6.3",
+        "vitest": "^0.34.6",
+        "react": "^18",
+        "react-dom": "^18"
+
```
Comment (SamuelAlev) [inline packages/platform-app/package.json:43]:
```suggestion
```

File: packages/platform-app/package.json [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "typecheck": "tsc --noEmit",
+        "test": "vitest run",
+        "test:watch": "vitest",
+    },
+    "devDependencies": {
+        "@frontify/app-bridge": "^3.1.0",
```
Comment (SamuelAlev) [inline packages/platform-app/package.json:31]:
```suggestion
        "@frontify/app-bridge": "workspace:^",
```

File: packages/platform-app/package.json (hunk 4 above)
Comment (julianiff) [reply packages/platform-app/package.json]: I tried a publish of the package to see if everything works and it complained with the workspace setting

Comment (SamuelAlev) [reply packages/platform-app/package.json:45]: With `pnpm publish`?

File: packages/platform-app/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Platform App Types
+
+## Code Structure
+Defines the App and gives typing for the settings
+
+```ts
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+import { defineApp } from '@frontify/platform-app';
```
Comment (SamuelAlev) [inline packages/platform-app/README.md:8]:
```suggestion
/* (c) Copyright Frontify Ltd., all rights reserved. */

import { defineApp } from '@frontify/platform-app';
```

File: packages/platform-app/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example
+
+```ts
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { defineSettings } from '@frontify/platform-app';
+
+export const settings = defineSettings({
+    Credentials: [
```
Comment (SamuelAlev) [inline packages/platform-app/README.md:29]:
```suggestion
    credentials: [
```
as it automatically title case the key

File: packages/platform-app/package.json [hunk 9]
```diff
@@ -0,0 +1,50 @@
+{
+    "name": "@frontify/platform-app",
+    "version": "0.0.3",
```
Comment (SamuelAlev) [inline packages/platform-app/package.json]:
```suggestion
    "version": "0.0.5",
```

File: packages/platform-app/package.json (hunk 4 above)
Comment (julianiff) [reply packages/platform-app/package.json:45]: used `npm publish`, but i also had an error in the package.json, ill change it back and try again 🦖

File: CODEOWNERS [hunk 10]
```diff
@@ -11,7 +11,7 @@ packages/app-bridge/src/errors/ValidationError.ts @Frontify/integration
 ## App Bridge Helpers
```
Comment (marguslant0) [inline CODEOWNERS:11]: Github complains about an error in codeowners - on line 2 `@Frontify/ecosystem` should now be `@Frontify/api-and-sdk`

File: packages/platform-app/src/platformApps.ts [hunk 11]
```diff
@@ -0,0 +1,58 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import type { AppBridgePlatformApp } from '@frontify/app-bridge';
+import type {
+    ChecklistBlock as ChecklistBlockSidebarSettings,
```
[the remark this answers is not in the document]
Comment (julianiff) [reply packages/platform-app/src/platformApps.ts]: sure sounds good.

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**